pages/form-builder/contactus.tsx

```diff
@@ -0,0 +1,281 @@
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { GetServerSideProps } from "next";
+
+import React, { useState } from "react";
+import Link from "next/link";
+import { Formik } from "formik";
+import { useTranslation } from "next-i18next";
+import * as Yup from "yup";
+import axios from "axios";
+import { logMessage } from "@lib/logger";
+import {
+  Button,
+  TextInput,
+  Label,
+  Alert,
+  ErrorListItem,
+  MultipleChoiceGroup,
+  TextArea,
+  Description,
+} from "@components/forms";
+import { StyledLink } from "@components/globals/StyledLink/StyledLink";
+
+export default function Contactus() {
+  const { t, i18n } = useTranslation(["form-builder", "common"]);
+  const [errorState, setErrorState] = useState({ message: "" });
```

Comment:
Hmmm if this state only has one key. It's probably best for it to be a string as opposed to an object. See the following 
https://beta.reactjs.org/learn/updating-objects-in-state

Reply:
Agreed on this just ""